docs/development.md

```diff
@@ -12,11 +12,8 @@ cd nativefier
 Install dependencies and build:
 
 ```bash
-# macOS and Linux
-npm run dev-up
-
-# Windows
-npm run dev-up-win
+npm install
```

Comment:
Do we not have to run `npm install` in the `app` directory? The `dev-up` script used to do it.

Reply:
> Do we not have to run `npm install` in the `app` directory? The `dev-up` script used to do it.

No, but this is still part broken and in flux. Currently I'm `npm install`ing in the `app` directory automatically through the `postinstall` script, which triggers, well, post-`npm install`. It works, but I noticed it causes https://github.com/npm/cli/issues/757 , which is a pretty terrible experience (60s freeze without progress). And it's not just for devs, but for users to, since I need these deps installed since webpack is no longer here to pack the whole electron app (our code plus dep code) into a single file packed in the npm artifact.

Thus, the first TODO in the list:

```
- [ ] Stopping to webpack `app` template caused https://github.com/npm/cli/issues/757 .
      Find a workaround or revert to `webpack`ing whole app + deps into single file.
```